cmd/tailscale/cli/file.go

```diff
@@ -286,22 +287,119 @@ func runCpTargets(ctx context.Context, args []string) error {
 	return nil
 }
 
+type existingFileBehavior int
+
+const (
+	FailOnExist         existingFileBehavior = iota
+	OverwriteExisting                        //  Overwrite any existing file at the target location
+	CreateNumberedFiles                      //  Create an alternately named file in the style of Chrome Downloads
+)
+
+var existingFileBehaviorNames = map[string]existingFileBehavior{
+	"abort":      FailOnExist,
+	"overwrite":  OverwriteExisting,
+	"autonumber": CreateNumberedFiles,
+}
+
+func (v *existingFileBehavior) String() string {
+	for k, e := range existingFileBehaviorNames {
+		if e == *v {
+			return k
+		}
+	}
+	return ""
+}
+
+func (v *existingFileBehavior) Set(s string) error {
+	if s == "" {
+		*v = FailOnExist
+		return nil
+	}
+	val, ok := existingFileBehaviorNames[strings.ToLower(s)]
+	if !ok {
+		return fmt.Errorf("'%s' not valid option for existing file behavior", s)
+	}
+	*v = val
+	return nil
+}
+
 var fileGetCmd = &ffcli.Command{
 	Name:       "get",
-	ShortUsage: "file get [--wait] [--verbose] <target-directory>",
+	ShortUsage: "file get [--wait] [--verbose] [--on_existing=(abort|overwrite|autonumber)] <target-directory>",
 	ShortHelp:  "Move files out of the Tailscale file inbox",
 	Exec:       runFileGet,
 	FlagSet: (func() *flag.FlagSet {
 		fs := newFlagSet("get")
 		fs.BoolVar(&getArgs.wait, "wait", false, "wait for a file to arrive if inbox is empty")
 		fs.BoolVar(&getArgs.verbose, "verbose", false, "verbose output")
+		fs.Var(&getArgs.onExisting, "on_existing", "behavior when target file already exists")
```

Comment:
how about "conflict" instead of "on_existing"

 --conflict={abort,overwrite,rename}

?